CHANGES.rst

```diff
@@ -2,14 +2,19 @@
 Changelog
 =========
 
-Version 3.0.1 (2019-XX-XX)
+Version 3.1.0 (2019-XX-XX)
 ==========================
 
 - fix ``getargspec`` ``DeprecationWarning``
 - fix ``FutureWarning`` in ``filter_array_like``
 - remove ``funcsigs`` requirement
 - fix ``_apply_partition_key_predicates`` ``FutureWarning``
 
+**Semi-breaking:**
```

Comment:
First of all, there is no such thing as a semi breaking change :) I think we should write up something about what we actually guarantee to be stable across releases and what not (e.g. specfication is tracked using metadata version, not the package version)

That being said. Is it actually breaking at all? Isn't the make_meta calls normalize_type for all columns so we just shifted the logic.

Reply:
@fjetter in our use case, it is not breaking but `normalize_type` is public API and it normalizes more types now so strictly speaking, this is a breaking change

Reply:
I'm just having issues with the term semi-breaking but that's not a showstopper.